ui/src/app/machines/views/MachineDetails/MachineTests/MachineTests.tsx

```diff
@@ -93,9 +93,15 @@ const MachineTests = (): JSX.Element => {
                 string,
                 ScriptResult[]
               ]) => {
+                const { model, name, serial } =
+                  scriptResults[0]?.parameters?.storage?.value || {};
                 return (
                   <div key={physical_blockdevice}>
-                    <h5 data-test="storage-heading">{physical_blockdevice}</h5>
+                    <h5 data-test="storage-heading">
+                      {model && name && serial
+                        ? `/dev/${name} (Model: ${model}, Serial: ${serial})`
```

Comment:
I know it's how it was in legacy, but I think `model:` and `serial:` should be lowercase shouldn't they?